### PR TITLE
Add override sample click_to_google_ntp

### PR DESCRIPTION
**Pull request by jhauga**
Added directory _`click_to_google_ntp`_ to `api-samples` in `override` directory. Builds on **`override/blank_ntp`**, demonstrating the customizable feature of the new tab page.

To see a working demo of sample visit GitHub page of _`Ctrl + click`_ [click_to_google.html](https://jhauga.github.io/chrome-ext-sample-click_to_google_ntp/).

### CODE REVIEW

File: api-samples/override/click_to_google_ntp/click_to_google.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
+<html>
+  <head>
+    <title>New Tab</title>
+    <style>
```
Comment (jhauga) [inline api-samples/override/click_to_google_ntp/click_to_google.html:4]: Demonstrates internal CSS with custom html page.

File: api-samples/override/click_to_google_ntp/click_to_google.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      }
+    </style>
+  </head>
+
+  <body>
+    <div id="textElement">Click<small>(<em>anywhere</em>)</small> to:</div>
+    <img id="chromeIcon" src="chrome_icon.png" alt="chrome icon" />
+    <img id="googleIcon" src="google_icon.svg" alt="chrome icon" />
+
+    <script src="script.js"></script>
```
Comment (jhauga) [inline api-samples/override/click_to_google_ntp/click_to_google.html:115]: Demonstrate linking to external JavaScript with custom html "New Tab Page".

File: api-samples/override/click_to_google_ntp/click_to_google.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      div#textElement * {
+        animation-name: three;
+        animation-duration: 3s;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div id="textElement">Click<small>(<em>anywhere</em>)</small> to:</div>
+    <img id="chromeIcon" src="chrome_icon.png" alt="chrome icon" />
```
Comment (jhauga) [inline api-samples/override/click_to_google_ntp/click_to_google.html:112]: Demonstrate linking to image with custom html "New Tab Page".

File: api-samples/override/click_to_google_ntp/click_to_google.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+      div#textElement,
+      div#textElement * {
+        animation-name: three;
+        animation-duration: 3s;
+      }
+    </style>
+  </head>
+
+  <body>
```
Comment (jhauga) [inline api-samples/override/click_to_google_ntp/click_to_google.html:110]: Demonstrate basic dynamic element with custom html "New Tab Page".

File: api-samples/override/click_to_google_ntp/click_to_google.html
Comment (jhauga) [inline api-samples/override/click_to_google_ntp/click_to_google.html:1]: Mainly this is an overview of how a custom html "New Tab Page" can be used with the fundamentals of web development.

File: api-samples/override/click_to_google_ntp/click_to_google_icon16.png
Comment (jhauga) [inline api-samples/override/click_to_google_ntp/click_to_google_icon16.png:1]: Working demo for implementing favicons.

File: api-samples/override/click_to_google_ntp/chrome_icon.png
Comment (jhauga) [inline api-samples/override/click_to_google_ntp/chrome_icon.png:1]: Support element for custom html "New Tab Page".

File: api-samples/override/click_to_google_ntp/google_icon.svg
Comment (jhauga) [inline api-samples/override/click_to_google_ntp/google_icon.svg:1]: Support element for custom html "New Tab Page".

File: api-samples/override/click_to_google_ntp/manifest.json
Comment (jhauga) [inline api-samples/override/click_to_google_ntp/manifest.json:1]: Shows basic use of adding favicons and incorporating custom html "New Tab Page" with fundamental elements of web development.

File: api-samples/override/click_to_google_ntp/script.js
Comment (jhauga) [inline api-samples/override/click_to_google_ntp/script.js:1]: Simple block for JavaScript demo.

File: package-lock.json
Comment (jhauga) [inline package-lock.json:1]: JSON file after following steps in [CONTRIBUTING.md](https://github.com/GoogleChrome/chrome-extensions-samples/blob/main/CONTRIBUTING.md#setting-up-your-environment).

**Review by jhauga — COMMENTED** [review]
Added comments to most of the new files to give a brief overview. Mainly this is a working sample of using a custom html "New Tab Page" that implements web development fundamentals. In addition the semantics of incorporating a custom favicon is demonstrated.

**Comment by oliverdunk** [comment]
Thanks for opening this - unfortunately I'm not sure that it demonstrates much beyond our [existing sample](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/api-samples/override/blank_ntp) so I don't think it makes sense to add another. I'd encourage you to keep building things though and perhaps you could put this on your own GitHub for people to find?

**Pull request closed without merging**